Source/Core/VideoCommon/AbstractTexture.cpp

```diff
@@ -26,6 +26,8 @@ bool AbstractTexture::Save(const std::string& filename, unsigned int level)
   // anyway, so this is fine for now.
   ASSERT(!IsCompressedFormat(m_config.format));
   ASSERT(level < m_config.levels);
+  // We can't copy from float (HDR) textures to RGBA8
```

Comment:
Curious, I know little about this space.  Is there a texture format that works to capture the HDR color space?

Reply:
I'm pretty sure that function would fail with any texture that isn't RGBA8, at least on DX?
But for sure it won't work with a float texture. To capture an HDR screenshot you can either tonemap it to SDR or save it to an .exr or something. It's not necessary really as this code is unrelated.